# Removes old -R flag from link line, as it's not supported un newer compilers.
# Just leave it up to the user if they want to rpath or not.
# Author: micketeer@gmail.com
diff -ru scripts.orig/libbrotlicommon.pc.in scripts/libbrotlicommon.pc.in
--- scripts.orig/libbrotlicommon.pc.in	2021-07-12 21:37:48.253345970 +0200
+++ scripts/libbrotlicommon.pc.in	2021-07-12 21:38:14.466282849 +0200
@@ -7,5 +7,5 @@
 URL: https://github.com/google/brotli
 Description: Brotli common dictionary library
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -R${libdir} -lbrotlicommon
+Libs: -L${libdir} -lbrotlicommon
 Cflags: -I${includedir}
diff -ru scripts.orig/libbrotlidec.pc.in scripts/libbrotlidec.pc.in
--- scripts.orig/libbrotlidec.pc.in	2021-07-12 21:37:48.252345973 +0200
+++ scripts/libbrotlidec.pc.in	2021-07-12 21:38:00.008317664 +0200
@@ -7,6 +7,6 @@
 URL: https://github.com/google/brotli
 Description: Brotli decoder library
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -R${libdir} -lbrotlidec
+Libs: -L${libdir} -lbrotlidec
 Requires.private: libbrotlicommon >= 1.0.2
 Cflags: -I${includedir}
diff -ru scripts.orig/libbrotlienc.pc.in scripts/libbrotlienc.pc.in
--- scripts.orig/libbrotlienc.pc.in	2021-07-12 21:37:48.252345973 +0200
+++ scripts/libbrotlienc.pc.in	2021-07-12 21:38:07.535299541 +0200
@@ -7,6 +7,6 @@
 URL: https://github.com/google/brotli
 Description: Brotli encoder library
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -R${libdir} -lbrotlienc
+Libs: -L${libdir} -lbrotlienc
 Requires.private: libbrotlicommon >= 1.0.2
 Cflags: -I${includedir}
